content/api/vision/coordinates.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Coordinates
+---
+
+[Polar coordinates](https://en.wikipedia.org/wiki/Polar_coordinate_system) are represented by a distance from a set point and an angle from a set direction. [Cartesian coordinates](https://en.wikipedia.org/wiki/Cartesian_coordinate_system) are ones that are defined using a unique set of values (typically _x_, _y_ and _z_) to specify a point in space.
+
+{{% notice note %}}
+All distances are measured in metres.
+{{% /notice %}}
+
+## Polar Coordinates
+The polar coordinates are taken from the position of the webcam.
```

Comment:
I'm not sure this sentence reads as we want it to. How about phrasing this more like the Cartesian coordinates description is; something like "The polar coordinates of a marker are relative to the view of the camera."?